src/pybind/mgr/cephadm/module.py

```diff
@@ -1068,8 +1068,12 @@ def update_failed_daemon_health_check(self) -> None:
             self.set_health_warning('CEPHADM_FAILED_DAEMON', f'{len(failed_daemons)} failed cephadm daemon(s)', len(
                 failed_daemons), failed_daemons)
 
-    def get_first_matching_network_ip(self, host: str, sspec: ServiceSpec) -> Optional[str]:
-        sspec_networks = sspec.networks
+    def get_first_matching_network_ip(self,
+                                      host: str,
+                                      sspec: ServiceSpec,
+                                      sspec_networks: Optional[List[str]] = None) -> Optional[str]:
```

Comment:
```suggestion
    def get_first_matching_network_ip(
        self,
        host: str,
        sspec: ServiceSpec,
        sspec_networks: Optional[List[str]] = None
    ) -> Optional[str]:
```
NIT, I've been using this format recently when transitioning function def lines into multiple lines.